src/main/java/li/cil/scannable/client/shader/Shaders.java

```diff
@@ -99,7 +103,8 @@ public void reload(final ResourceProvider provider) {
                     }
                 }, name, format);
             } catch (final Exception e) {
-                LOGGER.error(e);
+                e.printStackTrace();
+                //LOGGER.error(e);
```

Comment:
Was this for testing and can be reverted?

Reply:
Yes; I remember making this change while getting the shaders to load on Fabric. It's been reverted now.